.github/CONTRIBUTING.md

```diff
@@ -37,8 +32,7 @@ Contributing to Issues
 * If the issue is marked as [`Up-for-Grabs`][up-for-grabs],
   the PowerShell Maintainers are looking for help with the issue.
```

Comment:
You can fix this in another PR, but this doesn't have anything to do with this change.